kettle/deployment-staging.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   annotations:
-    iam.gke.io/gcp-service-account: kettle@k8s-gubernator.iam.gserviceaccount.com
+    iam.gke.io/gcp-service-account: kettle@kubernetes-public.iam.gserviceaccount.com
   name: kettle
```

Comment:
The namespace is not specified. Maybe we should specify it in the Makefile ?

Reply:
Added namespace explicitly here @ameukam thanks!